docs_src/style-guide.md

```diff
@@ -95,4 +95,12 @@ def dihedral(x:TensorImage,
     if k in [2,4,5,7]: x = x.flip(-2)
     if k in [3,5,6,7]: x = x.transpose(-1,-2)
     return x
-```
\ No newline at end of file
+```
+
+## Notes
+
+* Don't use `typing` library.
+* Arguments should have one indent relative to method definition.
+* Only `cls` and `self` should be on same line of method name and first argument of `@patch` like `x` on `dihedral` sample.
+* Closing parenthesis should be aligned with `def`.
+* Keyword arguments (`**kwags`) should not be documented. (NOTE: not needed for the sprint, but see if `@delegates` can be used to automatically document keyword args).
```

Comment:
Remove, I already can :)

Reply:
Spec the (NOTE)

Reply:
Q: "spec the NOTE" means remove the note?

Also what you mean with "remove, I already can"? I'm not so sure

Reply:
I ermoved the note! :)